model/cloud/openrouter.go

```diff
@@ -0,0 +1,51 @@
+package cloud
```

Comment:
cloud->provider

becaus this package will holds our providers.

Also, please move on the same level as "model", so we can do subpackages for models.

Reply:
With this change we have `provider.Provider`. Should I rename `provider.Interface`?

Reply:
provider.Provider is fine. We have it in language and model too. Internally we do that too.